lib/tasks/order_details.rake

```diff
@@ -28,6 +9,11 @@ namespace :order_details  do
     AutoLogout.new.perform
   end
 
+  desc "mark order details with reservation only reservations as complete"
+  task :expire_reservation_only => :environment do
```

Comment:
Can we do this as part of the same `expire_reservations` rake task? That way we don't need to add a new cron entry.